package.json

```diff
@@ -36,8 +36,8 @@
     }
   },
   "dependencies": {
-    "@apollo/react-hooks": "^3.1.3",
-    "@jahia/data-helper": "1.0.12",
+    "@apollo/react-hooks": "^4.0.0",
```

Comment:
It's probably better to move to official @apollo/client package as this project is deprecated https://github.com/apollographql/react-apollo.